src/agents/_run_impl.py

```diff
@@ -1008,6 +1009,9 @@ async def execute_handoffs(
                     new_items=tuple(new_step_items),
                     run_context=context_wrapper,
                 )
+
+            if input_filter and handoff_input_data is not None:
+                logger.debug("Filtering inputs for handoff")
```

Comment:
this debug logging may not be so useful. we may want to remove it or include more info